noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/JdbcNoiseMap.java

```diff
@@ -57,6 +57,16 @@
     // Soil areas are splited by the provided size in order to reduce the propagation time
     protected double groundSurfaceSplitSideLength = 200;
     protected int soundReflectionOrder = 2;
+
+    public boolean isBodyBarrier() {
```

Comment:
missing function description

Reply:
I never use it, it is only an automatic generation reminiscence...

Reply:
I need to remove it or you can do ?

Reply:
What ? Clean your own code :P